<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>primingDialog</class>
 <widget class="QDialog" name="primingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Priming Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Input</string>
       </property>
       <property name="configSection" stdset="0">
        <string notr="true">primingCalculator</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="BtVolumeLabel" name="label_beerVol">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="text">
           <string>Beer Volume</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_beerVol</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="BtVolumeEdit" name="lineEdit_beerVol">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Amount of beer to prime</string>
          </property>
          <property name="editField" stdset="0">
           <string notr="true">beerVol</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="BtTemperatureLabel" name="label_temp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Beer Temperature</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_temp</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="BtTemperatureEdit" name="lineEdit_temp">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Temp of the beer</string>
          </property>
          <property name="editField" stdset="0">
           <string notr="true">beerTemp</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_vols">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Desired Volumes</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_vols</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="BtGenericEdit" name="lineEdit_vols">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>How many volumes of CO2 you want (1 L CO2 @ STP per L beer)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QRadioButton" name="radioButton_glucMono">
          <property name="text">
           <string>Glucose Monohydrate (corn sugar)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QRadioButton" name="radioButton_gluc">
          <property name="text">
           <string>Anhydrous Glucose</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QRadioButton" name="radioButton_sucrose">
          <property name="text">
           <string>Sucrose (table sugar)</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QRadioButton" name="radioButton_dme">
          <property name="text">
           <string>Dry Malt Extract</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Output</string>
       </property>
       <property name="configSection" stdset="0">
        <string notr="true">primingCalculator</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="BtMassLabel" name="label_output">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="text">
           <string>Prime with</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_output</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="BtMassEdit" name="lineEdit_output">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>How much priming ingredient to use</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="editField" stdset="0">
           <string notr="true">primeMass</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BtTemperatureEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
   <slots>
    <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtTemperatureLabel</class>
   <extends>QLabel</extends>
   <header>BtLabel.h</header>
   <slots>
    <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtMassLabel</class>
   <extends>QLabel</extends>
   <header>BtLabel.h</header>
   <slots>
    <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
    <slot>popContextMenu(QPoint)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtMassEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
   <slots>
    <signal>textModified()</signal>
    <slot>lineChanged()</slot>
    <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtGenericEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>BtVolumeLabel</class>
   <extends>QLabel</extends>
   <header>BtLabel.h</header>
   <slots>
    <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtVolumeEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
   <slots>
    <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_beerVol</tabstop>
  <tabstop>lineEdit_temp</tabstop>
  <tabstop>lineEdit_vols</tabstop>
  <tabstop>radioButton_glucMono</tabstop>
  <tabstop>radioButton_gluc</tabstop>
  <tabstop>radioButton_sucrose</tabstop>
  <tabstop>radioButton_dme</tabstop>
  <tabstop>lineEdit_output</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>label_beerVol</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_beerVol</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_temp</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_temp</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_output</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_output</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
